src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs

```diff
@@ -38,16 +38,16 @@ public NuTexGenHdr Deserialize(BinaryReader reader, uint nuTexHdrVersion)
                 {
                     path = reader.ReadSized32NullTerminatedString();
 
-                    uint nuTexGenHdrUnknown1 = reader.ReadUInt32BigEndian();
+                    uint nuttype = reader.ReadUInt32BigEndian();
```

Comment:
What about 
```suggestion
                    uint nuTexGenHdrType = reader.ReadUInt32BigEndian();
```
?

Reply:
nuttype is the internal name for the attributes of the NuTexture and has nothing to do with NuTexGenHdr

Reply:
Then maybe 
```suggestion
                    uint nuTextureType = reader.ReadUInt32BigEndian();
```

That made the name more accurate following the C# convention too (same for other comments)